core/tests/protractor/accessibility.js

```diff
@@ -24,11 +24,13 @@ var users = require('../protractor_utils/users.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
 
 describe('screenreader and keyboard user accessibility features', function() {
+  var oppiaContentContainer = element(by.css('.oppia-content-container'));
+  var ERROR_MESSAGE = 'Element takes too long to load in';
```

Comment:
Can you make this more specific? For example, `Content container taking too long to load`?

Reply:
done